### PR TITLE
Support syncing Dialog360 Whatsapp templates

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/utils/whatsapp/tasks.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def refresh_whatsapp_templates():
 
     with r.lock("refresh_whatsapp_templates", 1800):
         # for every whatsapp channel
-        for channel in Channel.objects.filter(is_active=True, channel_type="WA"):
+        for channel in Channel.objects.filter(is_active=True, channel_type__in=["WA", "D3"]):
```
Comment (rowanseymour) [inline temba/utils/whatsapp/tasks.py:107]: it's not ideal for this module to have to know about WhatsApp channel types... how about..

 * This module has a list of WhatsApp channel types that starts off empty
 * And a method like `register_channel_type(ch_type: str)` which adds a channel type to the list
 * `ChannelType` had a bool property like `whatsapp_tasks`. Then when loading channel types, if that is set, call `register_channel_type`

That way new WhatsApp channel types can be added and people don't have to update the tasks here.

Comment (norkans7) [reply temba/utils/whatsapp/tasks.py:107]: Right, I'll add that.

Comment (norkans7) [reply temba/utils/whatsapp/tasks.py:107]: I see these 2 types are the only compatible with that template syncing

I can change that on that third one we support.

Zenvia does not use namespaces so its templates seem not compatible with the existing and that probably implicates rethinking the editor approach too

Comment (rowanseymour) [reply temba/utils/whatsapp/tasks.py:107]: Let's merge this and you can revisit this

**Pull request merged**